R/utilities.R

```diff
@@ -83,6 +84,8 @@ error_on_date <- function(date, expected_date, days = 7, warning = FALSE){
     else if(fail){
         stop(str_c("Date is more than ", days, " different from expected"))
     }
+  
+    return(days_late)
```

Comment:
Is there a reason you set this this up to not return `days_late` if is fails? I think it'd still be useful to know when a dashboard was last updated even if it's >30 days old / have that info in the `last_update` files!

Reply:
changed!